packages/expo/src/Expo.ts

```diff
@@ -1,154 +1,706 @@
-export { FaceDetector };
-export { FileSystem };
-export { Font };
-import * as GL from 'expo-gl';
-const GLView = GL.GLView;
-export { GL, GLView };
-import * as GoogleSignIn from 'expo-google-sign-in';
-export { GoogleSignIn };
-export { ImageManipulator };
-export { Haptics };
-import * as ImagePicker from 'expo-image-picker';
-export { ImagePicker };
-export { LocalAuthentication };
-export { IntentLauncher };
-import * as Localization from 'expo-localization';
-export { Localization };
-import * as Crypto from 'expo-crypto';
-export { Crypto };
-export { Location };
-import * as MediaLibrary from 'expo-media-library';
-export { MediaLibrary };
-import * as Permissions from 'expo-permissions';
-export { Permissions };
-import * as Print from 'expo-print';
-export { Print };
-export { Sensors };
-export { SQLite } from 'expo-sqlite';
-export { SMS };
-export { Speech };
-export { TaskManager };
-export { GestureHandler } from './GestureHandler';
-export { default as MapView } from './Maps/MapView';
+
 export { AR };
-export { Amplitude };
-export { default as AuthSession } from './AuthSession';
-export { Brightness };
-export { default as DangerZone } from './DangerZone';
-export { default as ErrorRecovery } from './ErrorRecovery/ErrorRecovery';
-export { Facebook };
 export { Google };
-import * as Random from 'expo-random';
-export { default as Icon } from './Icon';
-export { Random };
-export { Sharing };
-export { default as KeepAwake, activate, deactivate } from 'expo-keep-awake';
-export { default as Linking } from './Linking/Linking';
-export { MailComposer };
-export { default as Notifications } from './Notifications/Notifications';
 export { ScreenOrientation };
-export { SecureStore };
+export { SplashScreen };
 export { StoreReview };
-export { default as Svg } from './Svg';
 export { Updates };
-export { WebBrowser };
+
 export { default as apisAreAvailable } from './apisAreAvailable';
-export { default as takeSnapshotAsync } from './takeSnapshotAsync/takeSnapshotAsync';
-const { Audio, Video } = AV;
-export { AV, Audio, Video };
-export { BlurView };
-export { LinearGradient } from 'expo-linear-gradient';
-export { FacebookAds };
 export { default as AppLoading } from './launch/AppLoading';
-export { SplashScreen };
-export { default as registerRootComponent } from './launch/registerRootComponent';
+export { default as AuthSession } from './AuthSession';
+export { default as DangerZone } from './DangerZone';
+export { default as ErrorRecovery } from './ErrorRecovery/ErrorRecovery';
+export { default as Linking } from './Linking/Linking';
 export { default as Logs } from './logs/Logs';
+export { default as Notifications } from './Notifications/Notifications';
 export { default as Pedometer } from './Pedometer';
-export { WebView } from './WebView';
+export { default as registerRootComponent } from './launch/registerRootComponent';
+export { default as takeSnapshotAsync } from './takeSnapshotAsync/takeSnapshotAsync';
```

Comment:
`takeSnapshotAsync` is not deprecated at the moment because it has additional ReactRef support, so `react-native-view-shot` is not a drop in replacement: https://github.com/expo/expo/issues/4042